bpf/common/dns.h

```diff
@@ -97,6 +97,37 @@ static __always_inline u8 is_dns(connection_info_t *conn) {
     return is_dns_port(conn->s_port) || is_dns_port(conn->d_port);
 }
 
+static __always_inline void populate_dns_record(dns_req_t *req,
+                                                const pid_connection_info_t *p_conn,
+                                                const u16 orig_dport,
+                                                const u32 size,
+                                                const u8 qr,
+                                                const u16 id,
+                                                const conn_pid_t *conn_pid) {
+    __builtin_memcpy(&req->conn, &p_conn->conn, sizeof(connection_info_t));
+
+    req->flags = EVENT_DNS_REQUEST;
+    req->len = size;
+    req->dns_q = qr;
+    req->id = bpf_ntohs(id);
+    req->ts = bpf_ktime_get_ns();
+    req->tp.ts = bpf_ktime_get_ns();
```

Comment:
nit: use `req->ts` here?

Reply:
Nice catch. I need the req->tp.ts so that we can perform the "is transaction too long" check for finding a parent request, but, since I have the timestamp there I don't need another. I'm removing the req->ts field.